erpnext/accounts/report/bank_reconciliation_statement/bank_reconciliation_statement.py

```diff
@@ -58,8 +58,8 @@ def get_columns():
 		{
 			"fieldname": "payment_entry",
 			"label": _("Payment Entry"),
-			"fieldtype": "Dynamic Link",
-			"options": "payment_document",
+			"fieldtype": "Link",
+			"options": "Payment Entry",
```

Comment:
The column also shows Journal Entry, hence keep it Dynamic Link, just add another column 'Payment Document' to fix link issue.

Reply:
Added changes